python/python/bystro/proteomics/fragpipe_utils.py

```diff
@@ -0,0 +1,21 @@
+"""Provide various utilities for Fragpipe analysis."""
```

Comment:
nit: Are these for Fragpipe analysis or for ingesting/processing Fragpipe output files?